frontend/src/components/Pages/Home/index.js

```diff
@@ -14,7 +14,7 @@ const Home = () => {
 
     return (
         <React.Fragment>
-            <Typography variant='h5' align='center' component='h2' style={{ margin: '20px' }}>Make your own personalised style for the Quotes</Typography>
+            <Typography variant='h5' align='center' component='h2' style={{ margin: '20px', padding: '25px' }}>Make your own personalised style for the Quotes</Typography>
```

Comment:
LGTM, can you revert the package-lock.json?

Reply:
sure

Reply:
It is done